config/recipe-hyprland.yml

```diff
@@ -28,6 +28,37 @@ modules:
       - hyprland
       - waybar
       - xdg-desktop-portal-hyprland
+      - xorg-x11-server-Xwayland
+
+      # wayland utilities
+      - swaybg
```

Comment:
should these be common packages?

Reply:
Looks like Xwayland is in common, but swaybg is currently not